package/yast2-core.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Mon Nov  7 12:51:20 UTC 2016 - jreidinger@suse.com
+
+- ag_any: Fix early exit when optional element read EOL like e.g
```

Comment:
Hey, EOL is not EOF!
> ag_anyangent: Do not fail the parse when an Optional syntax meets EOF,
e.g. for a fstab without a trailing newline (bsc#429326#c11)